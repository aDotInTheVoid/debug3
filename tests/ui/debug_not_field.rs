struct NoDebug;

#[derive(debug3::Debug)]
struct HasDebug(NoDebug);

fn main() {}
