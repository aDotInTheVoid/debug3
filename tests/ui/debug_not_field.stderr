error[E0277]: the trait bound `NoDebug: debug3::Debug` is not satisfied
 --> tests/ui/debug_not_field.rs:3:10
  |
3 | #[derive(debug3::Debug)]
  |          ^^^^^^^^^^^^^ the trait `debug3::Debug` is not implemented for `NoDebug`
  |
  = note: required for the cast to the object type `dyn debug3::Debug`
  = note: this error originates in the derive macro `debug3::Debug` (in Nightly builds, run with -Z macro-backtrace for more info)
